src/Hint/NoCapitalisms.hs

```diff
@@ -0,0 +1,113 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
+{-
+    Detect uses of capitalisms
+
+    Only allow up to two consecutive capital letters in identifiers.
```

Comment:
What is the reason for this exception? `ID`? Personally I'd prefer uniformity here.

Reply:
That would be one case.

I'm all for uniformity, by I'm afraid not allowing this exception could make this rule very annoying. Let me do some statistics on you internal code base…